components/organisms/UserPortfolio.tsx

```diff
@@ -61,3 +63,33 @@ const YourLoans = () => {
         <TokenRowYourLoans address={"0x..."} name={"USD Coin"} apy={"5.57"} balance={"3.26"} />
     </>
 };
+
+const Styles = StyleSheet.create({
+    container: {
+        flexDirection: "row",
+    },
+    view: {
```

Comment:
`view` is so generic. Let's use something like `tabs` or `tabsContainer`, opt to you